/**:
  ros__parameters:
    # see https://github.com/open-mmlab/mmdetection3d/blob/fe25f7a51d36e3702f961e198894580d83c4387b/mmdet3d/evaluation/metrics/nuscenes_metric.py#L386-L393
    camera_order_remapping:
      0: 0 # CAMERA_FRONT
      1: 3 # CAMERA_BACK
      2: 2 # CAMERA_FRONT_LEFT
      3: 4 # CAMERA_BACK_LEFT
      4: 5 # CAMERA_BACK_RIGHT
      5: 1 # CAMERA_FRONT_RIGHT
    temporary_params:
      bins_directory_path: /home/minoda/ghq/github.com/exiawsh/StreamPETR/work_dirs/intermediate/
    model_params:
      engine_backbone_path: "$(var model_path)/simplify_extract_img_feat.engine"
      engine_head_path: "$(var model_path)/simplify_pts_head_memory.engine"
      engine_position_embedding_path: "$(var model_path)/simplify_position_embedding.engine"
      precision_backbone: "fp32"
      precision_head: "fp32"
      precision_position_embedding: "fp32"
      point_cloud_range: [-61.2, -61.2, -10.0, 61.2, 61.2, 10.0]
      input_image_height: 256
      input_image_width: 704
    post_process_params:
      confidence_threshold: 0.5
      # circle_nms_dist_threshold: 0.5
      iou_nms_target_class_names: ["CAR"]
      iou_nms_search_distance_2d: 10.0
      iou_nms_threshold: 0.1
      # yaw_norm_thresholds: [0.3, 0.3, 0.3, 0.3, 0.0]